watsong/spotify.py

```diff
@@ -299,6 +290,21 @@ def create_playlist(
     )
 
 
+def logout() -> str:
+    """
+    Change who you are logged in as.
+    Return "" on successful change, else return a string for the error
+    """
+    cache_file_path = f"{os.getcwd()}\\.cache"
```

Comment:
Use this for platform-independence: https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
Thanks, good catch!